src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/Batch/BatchTests.cs

```diff
@@ -12,6 +12,7 @@ namespace Microsoft.Data.SqlClient.ManualTesting.Tests
 {
     public static class BatchTests
     {
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
```

Comment:
NIT, skipping the target framework doesn't seem necessary since this file only included with net6.0 compatible versions in the csproj file.

Reply:
removed them all in this file